.gitignore

```diff
@@ -27,6 +27,8 @@ dlldata.c
 *.md.sub
 user_docs/*/*.html
 user_docs/*/*.css
+user_docs/*/*.md
+!user_docs/en/*.md
```

Comment:
I don't think this is safe yet - at least not until we fully switch to .xliff.
We still have some commits we need to drag over from https://github.com/nvaccess/nvda-l10n

```suggestion
```

Reply:
we also need to be able to delete the *.md files which are currently committed when they migrate xliff